packages/pkgpanda-api/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p $PKG_PATH/etc/
+
+cat <<EOF > $PKG_PATH/etc/pkgpanda-api.conf.py
```

Comment:
nit: What value is having this config file and passing it through giving us currently? Seems like a lot of work which could just be replaced by a constant for now.